app/src/main/java/com/spoony/spoony/presentation/userpage/mypage/MyPageViewModel.kt

```diff
@@ -15,8 +18,14 @@ class MyPageViewModel @Inject constructor(
     private val authRepository: AuthRepository
 ) : ViewModel() {
 
-    private val _state: MutableStateFlow<MyPageState> = MutableStateFlow(MyPageState())
-    val state: StateFlow<MyPageState>
+    private val _state: MutableStateFlow<UserPageState> = MutableStateFlow(
+        UserPageState(
+            userType = UserType.MY_PAGE,
+            profile = UserProfile(),
+            spoonCount = 0
+        )
+    )
+    val state: StateFlow<UserPageState>
         get() = _state
```

Comment:
```suggestion
    val state: StateFlow<UserPageState>
        get() = _state.asStateFlow()
```
P1: 퀴즈입니당 두 코드의 차이점은 무엇일까요?

Reply:
두번째 방식이 StateFlow래퍼로 감싸서 더 강한 읽기 전용을 명시하네요 몰랐음ㄷㄷ
UserPage도 같은 방식으로 수정하겠습니다~

Reply:
키키 좋아욤 추가로 읽기 전용으로 명시 안하면 외부에서 다시 mutableStateFlow로 캐스팅해서 데이터 수정하는 것이 가능해져요..!!